install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2857,14 +2857,9 @@ clustermesh:
       pullPolicy: "${PULL_POLICY}"
 
     etcd:
-      # -- Clustermesh API server etcd image.
-      image:
-        override: ~
-        repository: "${ETCD_REPO}"
-        tag: "${ETCD_VERSION}"
-        digest: "${ETCD_DIGEST}"
-        useDigest: true
-        pullPolicy: "${PULL_POLICY}"
+      # The etcd binary is included in the clustermesh API server image, so the same image from above is reused.
+      # Independent override isn't supported, because the clustermesh-apiserver expects and is tested against the etcd
```

Comment:
nit: expects what?

Reply:
I could reword this to be clearer I think. I mean to say that a specific clustermesh-apiserver is tested against a specific etcd version. So it expects that etcd version.

Reply:
Fixed.